taipy/templates/setup.py

```diff
@@ -23,7 +23,7 @@
     if vext := version.get("ext"):
         version_string = f"{version_string}.{vext}"
 
-test_requirements = ["pytest>=3.8"]
+test_requirements = ["pytest>=6.0"]
```

Comment:
Why?
1. This should be aligned with versions of the other sub-packages. They are still in 3.8.
2. Apparently, it is not used for the `templates` sub-packages. It could be a bug.
3. It should be done in another PR if we want to change it.

Reply:
👍